README.md

```diff
@@ -28,10 +59,19 @@ This repo contains 2 types of issue templates:
 
 This special repository also contains a special file `./profile/README.md` that will appear on the organizations profile.
 
+## Contributing
+
+To contribute to this repository, please see the [contribution guidelines](CONTRIBUTING.md).
+
 <!-- reference urls -->
 
-[gh-action-cvs]: github.com/plantemoran-ai-engineering/.github/actions/workflows/code-analysis.yml
-[gh-action-lint]: github.com/plantemoran-ai-engineering/.github/actions/workflows/linting.yml
+[developer-workstation-config-docs]: https://plantemoran-appdev.github.io/github-process-docs/workstation-setup/workstation-config/
+[gh-action--backup-repo]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/backup-repo.yml
+[gh-action-cvs]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/code-analysis.yml
+[gh-action-lint]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/linting.yml
 [gh-creating-default-comm-health-file]: https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file
-[img-gh-action-cvs-badge]: github.com/plantemoran-ai-engineering/.github/actions/workflows/code-analysis.yml/badge.svg
-[img-gh-action-lint-badge]: github.com/plantemoran-ai-engineering/.github/actions/workflows/linting.yml/badge.svg
+[gh-action--appscan-sast]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/hcl-appscan-sast-scan.yml
```

Comment:
Keep w/ a consistent naming pattern.
e.g. `gh-action--<action-name>`
(you have some w/ 1 dash and some w/ 2)

Here is how I am currently doing it:
```text
[gh-action--appscan-codesweep]
[gh-action--appscan-sast]
[gh-action--codeql]
[gh-action--linting]
etc.
```

Reply:
Good catch! Fixed.